internal/cli/cmd/clientsync.go

```diff
@@ -20,12 +20,11 @@ import (
 // NewClientSyncCmd implements the command: epinio client-sync
 func NewClientSyncCmd(client *usercmd.EpinioClient) *cobra.Command {
 	return &cobra.Command{
-		Use:   "client-sync",
-		Short: "Downloads a client binary matching the currently logged server",
-		Long:  `Synchronizes the epinio client with the server by downloading the matching binary and replacing the current one.`,
+		Use:          "client-sync",
+		Short:        "Downloads a client binary matching the currently logged server",
+		SilenceUsage: true,
+		Long:         `Synchronizes the epinio client with the server by downloading the matching binary and replacing the current one.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cmd.SilenceUsage = true
```

Comment:
IIRC the location of the silenceUsage makes a difference when it applies.
As changed it applies even to argument processing errors (not enough args, etc.)
Used in the command, it silences the usage only for errors coming from the command itself, and not for argument issues thrown by cobra itself.

As a side note, the command is missing a spec telling that it takes 0 arguments.
I suspect that due to the missing spec argument errors currently cannot happen, as it always any number of arguments which are then silently ignored.

Reply:
Confirmed using `epinio info` that my recollection is correct.

Added the `ExactArgs(0)` to the `info` command, and then ran it with `Silence` in the command and in the structure. Result:

In the command, as before the change in placement was made. :heavy_check_mark: 
```
work@tagetarl:~/SUSE/dev/epinio> ep info 9034290 902185r93401
Usage:
  epinio info [flags]

Flags:
  -h, --help   help for info

Global Flags:
  -H, --header stringArray       Add custom header to every request executed
  -c, --kubeconfig string        (KUBECONFIG) path to a kubeconfig, not required in-cluster
      --no-colors                Suppress colorized output
      --settings-file string     (EPINIO_SETTINGS) set path of settings file (default "/home/work/.config/epinio/settings.yaml")
      --skip-ssl-verification    (SKIP_SSL_VERIFICATION) Skip the verification of TLS certificates
      --timeout-multiplier int   (EPINIO_TIMEOUT_MULTIPLIER) Multiply timeouts by this factor (default 1)
      --trace-file string        (TRACE_FILE) Print trace messages to the specified file
      --trace-level int          (TRACE_LEVEL) Only print trace messages at or above this level (0 to 255, default 0, print nothing)
      --trace-output string      (TRACE_OUTPUT) Sets trace output format [text,json] (default "text")
      --verbosity int            (VERBOSITY) Only print progress messages at or above this level (0 or 1, default 0)


❌  accepts 0 arg(s), received 2
```

In the structure, i.e. per the change above. :boom: , not desired.
```
work@tagetarl:~/SUSE/dev/epinio> ep info 9034290 902185r93401

❌  accepts 0 arg(s), received 2
```

Reply:
Ah, you are right, I forgot about this. Thanks. Fixing this.